lib/src/main/kotlin/bstrees/balancers/RBBalancer.kt

```diff
@@ -93,7 +94,129 @@ class RBBalancer<T : Comparable<T>> : TreeBalancer<T, RBNode<T>> {
         return currentNode
     }
 
+    /** Accepts the node to be deleted. Returns new tree root */
```

Comment:
Please specify that `balanceAfterDeletion` doesn't itself delete `node` and caller must delete (reassign parent's child basically) the node on their own